script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -1,7 +1,7 @@
 [sha:1e2bbef0a649bb39ef95e2f650878c513c817c11]{
 -- Page 10002, Offset 14041.
 -- 「……そうだな。それが一番いいと思う」
-"...Yeah. I think that's the best way to do it."
+"...That's right. I think it's the best way to do it."
```

Comment:
"I think _that's_ the best..." is correct form here. He's directly referring back to the choice sentence. 
Yeah also reads better than "that's right" to me here.